azure/template.json

```diff
@@ -319,6 +319,22 @@
       "metadata": {
       "description": "Configure Number of database server CPU"
       }
+    },
+  "actionGroupName": {
+      "type": "string",
+      "defaultValue": "",
+      "minLength": 1,
+      "metadata": {
+          "description": "Name of ActionGroup to send notification"
+      }
+  },
+  "appServicePlanResourceGroup": {
```

Comment:
Do we need a new parameter?
Can't we use resourcegroup() funstion like here: https://github.com/DFE-Digital/teacher-training-api/blob/62dc4e4f40e18a3f3dafcc13b2436169f9dd4407/azure/template.json#L494 ?

Reply:
done....